### ISSUE TITLE
Build failure: quodlibet-full

### ISSUE THREAD

**Issue by 9p4** [NixOS/nixpkgs#309588]
### Steps To Reproduce

1. build quodlibet-full

### Build log

https://gist.github.com/9p4/03c3fbd9a66b56b86b405f2cad449e91

### Additional context

N/A

### Notify maintainers

@coroa @pbogdan 

### Metadata

Please run `nix run nixpkgs#nix-info -- -m` and paste the result.

```console
[user@system:~]$ nix run nixpkgs#nix-info -- -m
 - system: `"x86_64-linux"`
 - host os: `Linux 6.8.8, NixOS, 24.05 (Uakari), 24.05.20240502.63c3a29`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.2`
 - channels(root): `""`
 - nixpkgs: `/nix/store/p69bcs7ma6ijj8v9xsrg3nq3nn8ryn95-source`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by SFrijters** [comment]
Ran into this as well earlier today, I'm currently trying to bisect https://hydra.nixos.org/build/258311308, but it's taking a while...

**Comment by 9p4** [comment]
Temporarily have it working: https://git.sr.ht/~fd/nix-configs/commit/623eb6046f639ac58248a76d7bccce0c4b75f03d (do not do this in prod).

**Comment by SFrijters** [comment]
Bisected to 676ca4e68f512dadba80fb87893ea6e1e4d3e116 (gdk-pixbuf: 2.42.10 -> 2.42.11), currently rebuilding with https://github.com/NixOS/nixpkgs/pull/308976 to see if that fixes this issue as well.

**Comment by SFrijters** [comment]
Can confirm that the overlay linked by @9p4 is a workaround and that https://github.com/NixOS/nixpkgs/pull/308976 fixes the issue (but it causes many heavy rebuilds so I personally wouldn't apply it as an overlay).

**Comment by 9p4** [comment]
Closing as https://github.com/NixOS/nixpkgs/pull/308976 was merged.

**Comment by qbit** [comment]
This is still broken on unstable, should 104f6678f23088e2d9e2ea870aa0bc3cb3bf4ce2 be cherry picked to master?

**Comment by SFrijters** [comment]
It's included in https://github.com/NixOS/nixpkgs/pull/312133 so it shouldn't be long until the fix hits master (I hope) since it is supposed to be merged before 24.05.
